package.json

```diff
@@ -4,7 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prestart": "npm run build",
```

Comment:
Please create a `dev` script that will run the app during development using [Nodemon](https://nodemon.io/) and remove this `prestart` script. This is because it will be automatically run before every `start` which will cause issues on Heroku when it runs after the dev dependencies have been pruned.

Reply:
Ok
Will do that immediately

Reply:
It has been changed